test/integration/02-user-wallet/02-send-onchain.spec.ts

```diff
@@ -926,3 +901,89 @@ describe("UserWallet - onChainPay", () => {
     expect(status).toBeInstanceOf(LessThanDustThresholdError)
   })
 })
+
+describe("UsdWallet - onChainPay", () => {
+  describe("to an internal address", () => {
+    it("sends from usd wallet to usd wallet", async () => {
+      const res = await testInternalSend({
+        senderAccount: accountB,
+        senderWalletId: walletIdUsdB,
+        recipientWalletId: walletIdUsdA,
+        senderAmount: usdAmount,
+      })
+      if (res instanceof Error) throw res
+    })
+
+    it("sends from usd wallet to btc wallet", async () => {
+      const res = await testInternalSend({
+        senderAccount: accountB,
+        senderWalletId: walletIdUsdB,
+        recipientWalletId: walletIdA,
+        senderAmount: usdAmount,
+      })
+      if (res instanceof Error) throw res
+    })
+
+    it("sends from btc wallet to usd wallet", async () => {
+      const res = await testInternalSend({
+        senderAccount: accountA,
+        senderWalletId: walletIdA,
+        recipientWalletId: walletIdUsdB,
+        senderAmount: amount,
+      })
+      if (res instanceof Error) throw res
```

Comment:
what are you testing here? test should have expects if you are hiding it the helper function then is not a good practice.

Reply:
This is to de-duplicate the tests that run in that helper function. The function is re-used 8 times in this file. I'm not sure of better ways to de-duplicate these here, but open to suggestions.